provider/src/test/kotlin/org/gradle/kotlin/dsl/accessors/TypeAccessibilityProviderTest.kt

```diff
@@ -2,6 +2,8 @@ package org.gradle.kotlin.dsl.accessors
 
 import org.gradle.internal.classpath.ClassPath
 
+import org.gradle.kotlin.dsl.typeOf
```

Comment:
:o: Unnecessary addition?

Reply:
Nope, it moved.
See e16e0c6d2b07dd964293fa20246c3c8a80c40e2c

Reply:
Okay.